src/transformers/generation/logits_process.py

```diff
@@ -45,7 +45,7 @@ class LogitsProcessor:
     """Abstract base class for all logit processors that can be applied during generation."""
 
     @add_start_docstrings(LOGITS_PROCESSOR_INPUTS_DOCSTRING)
-    def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> torch.FloatTensor:
+    def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor, cur_len: int) -> torch.FloatTensor:
```

Comment:
Don't forget to update `LOGITS_PROCESSOR_INPUTS_DOCSTRING` accordingly (`LOGITS_PROCESSOR_INPUTS_DOCSTRING` is defined a few lines above)